packages/generator/src/edmx-parser/legacy/common/edmx-to-vdm.ts

```diff
@@ -188,7 +189,7 @@ const propertyFieldType = (type: string): string | undefined =>
 const propertyJsType = (type: string): string | undefined =>
   type.startsWith('Edm.') ? edmToTsType(type) : undefined;
 
-const complexTypeName = (type: string) => last(type.split('.'));
+export const complexTypeName = (type: string) => last(type.split('.'));
```

Comment:
Good catch. I moved the method to the util and for the generator package it is not complaining now: 
![image](https://user-images.githubusercontent.com/56544999/89778799-680e1580-db0e-11ea-9d18-fbae91e05a7d.png)

However, I think we should try to remove all circular dependencies so that we can execute this check in the pipeline to ensure quality over time. We can not rely on somebody seeing it in the review